nixos/lib/test-driver/test_driver/driver.py

```diff
@@ -10,6 +11,20 @@
 from test_driver.polling_condition import PollingCondition
 
 
+@contextmanager
+def must_raise(
```

Comment:
it looks like this isn't used anywhere, or ist it just too early in the morning for me?

Reply:
No it was too late at night last night I guess. I packaged all prerequisites for the I wrote for #155622 into one pr, should've mentioned that. I also wrote a similar local function in #154168, and it seems like it would be more generally useful so I thought it would be best to just provide it within the test driver